### DISCUSSION TITLE
Teach me

### DISCUSSION THREAD

**Discussion by trad1983** [community/community#55776] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I looking for some one teach me all about Aurelia frameworks online course please if anyone can help send me about price and contact details

**Comment by queenofcorgis** [comment]
Hi @trad1983 , we're happy you're here 😀

However, we currently do not allow job listings on the Community on Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

Thanks for your interest 👍
